os/initsystem/systemd.go

```diff
@@ -67,7 +66,7 @@ func (i Systemd) ServiceEnvironmentContent(env map[string]string) string {
 	var b strings.Builder
 	fmt.Fprintln(&b, "[Service]")
 	for k, v := range env {
-		_, _ = fmt.Fprintf(&b, "Environment=%s=%s\n", k, strconv.Quote(v))
+		_, _ = fmt.Fprintf(&b, "Environment=%s=%s\n", k, v)
```

Comment:
Should probably have been something along those lines:

```go
env := fmt.Sprintf("%s=%s", k, v)
_, _ = fmt.Fprintf(&b, "Environment=%s\n", strconv.Quote(env))
```

https://www.man7.org/linux/man-pages/man7/systemd.syntax.7.html#QUOTING